src/registry/r2.ts

```diff
@@ -131,11 +131,12 @@ export class R2Registry implements Registry {
     return checkManifestResponse;
   }
 
-  async listRepositories(limit?: number): Promise<RegistryError | ListRepositoriesResponse> {
+  async listRepositories(limit?: number, last?: string): Promise<RegistryError | ListRepositoriesResponse> {
     const env = this.env;
-    const options = {
+    const options:R2ListOptions = {
```

Comment:
nit: This is our bad due to not setting it up in the repo, but could we have the file linted with prettier?

Reply:
Sure!